### DISCUSSION TITLE
Mircrosoft Visual C++ 14.0 or greater is required on Windows

### DISCUSSION THREAD

**Discussion by Viator99** [qpsolvers/qpsolvers#257] [Q&A] [answered]
i'm becoming mad, pls help me
![Screenshot (169)](https://user-images.githubusercontent.com/45045229/99253447-11049480-2811-11eb-9410-fc99deb547f7.png)

**Comment by tytcc** [comment, marked as answer]
It seems that you need  install Mircrosoft Visual C++ 14.0 or greater. May be this can be helpful to you:
https://wiki.python.org/moin/WindowsCompilers#Microsoft_Visual_C.2B-.2B-_14.2_standalone:_Build_Tools_for_Visual_Studio_2019_.28x86.2C_x64.2C_ARM.2C_ARM64.29

**Comment by Viator99** [comment]
> 
> 
> It seems that you need install Mircrosoft Visual C++ 14.0 or greater. May be this can be helpful to you:
> https://wiki.python.org/moin/WindowsCompilers#Microsoft_Visual_C.2B-.2B-_14.2_standalone:_Build_Tools_for_Visual_Studio_2019_.28x86.2C_x64.2C_ARM.2C_ARM64.29

thank you so much, it is installed now

**Comment by stephane-caron** [comment]
In case the above links become outdated: for this issue, ones needs to install the [Visual C++ Build Tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/) to build all package dependencies.
